rpc/src/v1/impls/mod.rs

```diff
@@ -97,3 +101,48 @@ fn sign_and_dispatch<C, M>(client: &C, miner: &M, request: TransactionRequest, s
 	trace!(target: "miner", "send_transaction: dispatching tx: {}", encode(&signed_transaction).to_vec().pretty());
 	dispatch_transaction(&*client, &*miner, signed_transaction)
 }
+
+mod error_codes {
+	// NOTE [ToDr] Codes from -32000 to -32099
+	pub const UNKNOWN_ERROR: i64 = -32002;
+	pub const TRANSACTION_ERROR: i64 = -32010;
+	pub const ACCOUNT_LOCKED: i64 = -32020;
```

Comment:
there are also some errors floating around in `impls/eth.rs` that can be consolidated here